components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -87,7 +87,7 @@ internal class DisplayToolbar(
             field = value
             setTrackingProtectionState(siteTrackingProtection)
         }
-
+    internal var displaySeparatorView = true
```

Comment:
How about using `false` as default to not change the default behavior?

Reply:
Sounds good!

Reply:
Done!